### PR TITLE
Build/ts eslints errors

### PR DESCRIPTION
**Pull request by anchung333**
### Description

This PR implements changes to provide a better experience for the developer; these changes include getting rid of prettier and eslint warnings and errors, as well as updating packages and identifying them as the correct type of dependency.

### CODE REVIEW

**Comment by anchung333** [comment]
There are 3 eslint-warnings left that I am having some trouble with; feedback would be appreciated. thanks.

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export const Checkbox = ({
     multiline={!!subtitle}
     onClick={() => onChange(!checked)}
   >
-    <AminoCheckbox checked={checked}>
+    <AminoCheckbox checked={checked} id="amino-checkbox">
```
Comment (goodpickle) [inline src/components/Checkbox/Checkbox.tsx:91]: An id needs to be [globally unique](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id). What do you think about dynamically creating an id & `htmlFor`?

File: src/components/Radio/RadioGroup.tsx [hunk 2]
```diff
@@ -32,6 +32,15 @@ export const RadioGroup = <T extends { label?: string; value?: string }>({
 
   const activeItem = items.find((_, i) => i === active);
 
+  const initial = items.findIndex(el => {
+    const value = itemValuePath ? el[itemValuePath] : el.value;
```
Comment (goodpickle) [inline src/components/Radio/RadioGroup.tsx:36]: What do you think about cleaning this up similar to the `Select` component by removing itemValuePath & itemLabelPath?

**Review by goodpickle — CHANGES_REQUESTED** [review]

File: src/components/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (anchung333) [reply src/components/Checkbox/Checkbox.tsx]: brain fart; you're right. good catch thanks!

File: src/components/Radio/RadioGroup.tsx (hunk 2 above)
Comment (anchung333) [reply src/components/Radio/RadioGroup.tsx]: done

**Review by goodpickle — APPROVED** [review]

**Pull request merged**